recipes/pyparallel/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "pyparallel" %}
+{% set version = "0.2.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b5550293af42a42d7b2e1ada1224d3c3ce2f09b80e85421820e068655908c611
+
+build:
+  number: 0
+  # Note: --no-deps is currently required due to https://github.com/conda/conda-build/issues/3254
+  # Once resolved, it should be removed.
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
```

Comment:
Looks like it has C code. Please add the `compiler` part from the example here (no need to include the comments).

ref: https://github.com/conda-forge/staged-recipes/blob/3d1ebd7a0b04a659d8cb940f20c46f309d931795/recipes/example/meta.yaml#L38-L41

Reply:
Strange the C code is not in the `sdist`, but it is in GitHub repo. Appears it it only needed for Windows. This may require some more thought to address correctly.

cc @conda-forge/staged-recipes

Reply:
@jakirkham I did some more research and it turns out that _at least_ another tool/driver, `giveio`, would be required to use the `simpleio.dll`, and that it's questionable whether both would actually work on a modern Windows system.

I would therefore like to drop or at least postpone Windows builds of the package for the time being. Do you think that would be alright? I would add a selector to only build on non-Windows systems.